### ISSUE TITLE
Some generated files are different than their commited versions

### ISSUE THREAD

**Issue by mradenovic** [prescottprue/generator-react-firebase#216]
**Describe the bug**
Some files generated by `yo` are different than their versions committed with `husky` hooks. If one wants to pull the bug fixes from a new generator version, it will introduce changes to those files.

I wonder if this has something to do with some `examples` files being reintroduced with every pull request.

**To Reproduce**
1. Generate new files with `yo react-firebase --skip-install`
2. Commit all files with git
3. Regenerate all files with `yo react-firebase --skip-install --force`
4. `git status`

```
On branch master
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   CONTRIBUTING.md
        modified:   README.md
        modified:   database.rules.json
        modified:   firebase.json
        modified:   functions/jsconfig.json
        modified:   jsconfig.json
        modified:   public/manifest.json
        modified:   src/utils/router.js

no changes added to commit (use "git add" and/or "git commit -a")
```

**Expected behavior/code**

```
On branch master
nothing to commit, working tree clean
```

**Possible Solution**

**Additional Context/Screenshots**

**Comment by prescottprue** [comment]
This may be due to lint settings - thanks for reaching out

**Comment by prescottprue** [comment]
Was fixed in v5.7.1 - reach out if things aren't working as you expect now
